repository/src/main/java/org/apache/atlas/repository/store/graph/v2/AtlasEntityStoreV2.java

```diff
@@ -1802,6 +1816,41 @@ private AtlasStruct getStarredDetailsStruct(String assetStarredBy, long assetSta
         return starredDetails;
     }
 
+    private void createQualifiedNameHierarchyField(AtlasEntity entity, AtlasVertex vertex) {
+        MetricRecorder metric = RequestContext.get().startMetricRecord("createQualifiedNameHierarchyField");
+        try {
+            if (vertex == null) {
```

Comment:
Also add a check for the type which are excluded like Readme, Link, GCT

Reply:
No need for that, will automatically get discarded if it don't follow the same pattern. This will be a too much of hardcoding.